source/contribute.rst

```diff
@@ -267,76 +347,67 @@ sense to revert a commit documenting a feature without also reverting
 the code for that feature. That means there should be only once commit
 with both the code and the docs here.
 
-Rebasing on upstream changes
-----------------------------
+Rebasing
+--------
 
-Next, merge or rebase your commit against **doctrine/2.11.x**. With your
-work done in a **local** topic branch, you'll want to assist upstream
-merge by rebasing your commits. You can either do this manually with
-``fetch`` then ``rebase``, or use the ``pull --rebase`` shortcut. You
-may encounter merge conflicts, which you should fix and then mark as
-fixed with ``add``, and then continue rebasing with
-``rebase --continue``. At any stage, you can abort the rebase with
-``rebase --abort`` unlike nasty merges which will leave files strewn
-everywhere.
+On upstream changes
```

Comment:
"on" or "onto"? Not a native speaker myself. Git CLI suggests `onto`.

Reply:
From `man git rebase`: "it may not be the best idea to keep rebasing on top of the upstream"